src/tests/dialogs/unblockDialogs/unblockDirectDeposit.test.ts

```diff
@@ -205,41 +206,62 @@ describe("Unblock Direct Deposit Step", () => {
 
         assert.strictEqual(reply.text, expectedAccountMsg);
       });
-      it("Should fail gracefully after 3 errors", async () => {
-        const sut = new UnblockDirectDepositStep();
-        const client = new DialogTestClient("test", sut, testData.initialData, [
-          new DialogTestLogger(console),
-        ]);
 
-        const steps = [
-          [null, i18n.__("unblock_direct_deposit_transit")],
-          [i18n.__("unblock_direct_deposit_transit_retry"), `hhh`],
-          ["hhh!", ""],
-        ];
-        const activity: Partial<Activity> = {
-          text: testData.steps[0][0],
-          locale: "en",
-        };
-        await client.sendActivity(activity);
-
-        await client.getNextReply();
-        const reply = await client.getNextReply();
-        console.log("test22333", reply);
```

Comment:
If this console is required, is it possible to add more detail to the output?

Reply:
I will delete console.log. it is not necessary